csss-site/src/events/frosh/urls.py

```diff
@@ -45,5 +45,9 @@
     url(r'^2022$', frosh2022.index, name="Frosh Week 2022 Entrance"),
     url(r'^2022/frosh$', frosh2022.frosh, name="Frosh Week 2022"),
     url(r'^2022/secret$', frosh2022.secret, name="Not A Secret"),
+    # url(r'^2023$', frosh2023.index, name="Frosh Week 2023 Entrance"),
+    # url(r'^2023/frosh$', frosh2023.frosh, name="Frosh Week 2023"),
+    url(r'^2023/secret$', frosh2023.secret, name="Not A Secret"),
```

Comment:
you did not add w3c tests for these two new pages 😄

Reply:
I was not aware I had to make any test cases? The wiki doesn't mention it.

Reply:
>  The wiki doesn't mention it. 

fuck, fair point 😅 look [here](https://github.com/CSSS/csss-site/blob/master/csss_site_w3c/events/frosh/tests.py) for reference

Reply:
no, you do `python -u manage.py test --testrunner xmlrunner.extra.djangotestrunner.XMLTestRunner`

Reply:
It's still not working, even after I installed the requirements.txt for `csss_site_w3c`

I don't know what values to set the following env vars:
- UNIT_TEST_URL
- W3C_TEST_RESULT_DIRECTORY

Reply:
😅 i'm sorry man, I think I need to make an issue to address this cause I completely forgot that folks might want to do this.

But in the meantime what I suggest is going to https://validator.w3.org/ and plug in the non-working pages into there
like https://validator.w3.org/nu/?doc=https%3A%2F%2Fdev.sfucsss.org%2FPR-592%2Fevents%2Ffrosh%2F2023%2Fsponsor

Reply:
sure thing, and thanks for the workaround!